src/Exceptionless.Core/Settings.cs

```diff
@@ -69,7 +69,11 @@ public class Settings {
 
         public int MetricsServerPort { get; private set; }
 
-        public bool EnableMetricsReporting { get; private set; }
+        public string MetricsReportingDatabase { get; private set; }
+
+        public string MetricsReportingStrategy { get; private set; }
```

Comment:
I would prefer that we not have a metrics reporting mode, but we just check for the existence of config settings in order of highest priority. Because I want the least amount of config settings as possible. So I want things to be inferred and maybe even use a connection string format for this grouping of metrics settings so that we could have a single setting that contains all of the metrics config

Reply:
Would be great to have one for queues and storage as well. That way we don't have to have a ton of settings for extra implementations.